README.md

```diff
@@ -29,16 +29,36 @@ Broadside.configure do |config|
       scale: 15,
       command: ['bundle', 'exec', 'rake', 'resque:work'],
       env_file: '../.env.production'
-    }
+    },
     staging_web: {
       scale: 1,
       command: ['bundle', 'exec', 'puma'],
       env_file: '../.env.staging'
-    }
+    },
     staging_worker: {
       scale: 1,
       command: ['bundle', 'exec', 'rake', 'resque:work'],
       env_file: '../.env.staging'
+    },
+    # Example with a task_definition and service configuration which you use to bootstrap a service and
+    # initial task definition
+    game_save_as_json_blob_stream: {
+      scale: 1,
+      command: ['java', '-cp', '*:.', 'path.to.MyClass'],
+      env_file: '.env.production',
+      service: {
```

Comment:
the one thing that kind of sucks here is that if you want to just use the defaults to bootstrap, you still have to specify `service: {}` and `task_definition: {}` (without the keys, and without their values being a hash, you won't be able to trigger a bootstrap).

maybe `bootstrap` should just use the defaults if the user doesn't specify any extra config and not check for the existence of `service:` and `task_definition`?

not the end of the world just kind of blah